### PR TITLE
3 clean up (WIP)

### PR DESCRIPTION
**Pull request by payamST**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 Our project welcomes external contributions. If you have an itch, please feel
 free to scratch it.
 
-To contribute code or documentation, please submit a **FIXME** [pull request](https://github.com/ibm/repo-template/pulls).
+To contribute code or documentation, please submit a [pull request](https://github.com/IBM/magkit-vanity-url/pulls).
```
Comment (jfrantzius) [inline CONTRIBUTING.md:5]: must be magkit-test

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 
 A good way to familiarize yourself with the codebase and contribution process is
-to look for and tackle low-hanging fruit in the **FIXME** [issue tracker](https://github.com/ibm/repo-template/issues).
+to look for and tackle low-hanging fruit in the [issue tracker](https://github.com/IBM/magkit-vanity-url/issues).
```
Comment (jfrantzius) [inline CONTRIBUTING.md:8]: must be magkit-test

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -14,22 +14,20 @@ cannot be accepted at all!**
 
 ### Proposing new features
 
-If you would like to implement a new feature, please **FIXME** [raise an issue](https://github.com/ibm/repo-template/issues)
+If you would like to implement a new feature, please [raise an issue](https://github.com/IBM/magkit-vanity-url/issues)
```
Comment (jfrantzius) [inline CONTRIBUTING.md:17]: magkit-test

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -14,22 +14,20 @@ cannot be accepted at all!**
...
 before sending a pull request so the feature can be discussed. This is to avoid
 you wasting your valuable time working on a feature that the project developers
 are not interested in accepting into the code base.
 
 ### Fixing bugs
 
-If you would like to fix a bug, please **FIXME** [raise an issue](https://github.com/ibm/repo-template/issues) before sending a
-pull request so it can be tracked.
+If you would like to fix a bug, please [raise an issue](https://github.com/IBM/magkit-vanity-url/issues) before sending a
```
Comment (jfrantzius) [inline CONTRIBUTING.md:24]: magkit-test

**Review by jfrantzius — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (payamST) [reply CONTRIBUTING.md]: Done!

File: CONTRIBUTING.md (hunk 3 above)
Comment (payamST) [reply CONTRIBUTING.md]: Done!

File: CONTRIBUTING.md (hunk 4 above)
Comment (payamST) [reply CONTRIBUTING.md]: Done!

File: CONTRIBUTING.md (hunk 2 above)
Comment (payamST) [reply CONTRIBUTING.md]: Done!

**Review by jfrantzius — DISMISSED** [review]

**Review by wolf-bubenik — APPROVED** [review]

**Pull request merged**